.github/label-pr-config.yml

```diff
@@ -0,0 +1,83 @@
+autolabeler:
+  - label: 'Type: documentation'
+    files:
+      - '*.md'
+    # Checks there is no word character (alphanumeric & underscore) before target words, or after to avoid accidental matches
+    title:
+      - '/(?<![\w])((doc)|(docs)|(documentation))(?![\w])/i'
+  - label: 'Type: fix'
+    branch:
+      - '/fix\/.+/'
+    title:
+      - '/(?<![\w])((fix)|(fixes)|(fixed)|(bug)|(bugfix)|(bug fix))(?![\w])/i'
+  - label: 'Type: feature'
+    branch:
+      - '/feature\/.+/'
+    title:
+      - '/(?<![\w])((feat)|(feature))(?![\w])/i'
+  - label: 'Type: enhancement'
+    branch:
+      - '/enhancement\/.+/'
+    title:
+      - '/(?<![\w])((enhancement)|(enhancements))(?![\w])/i'
+  - label: 'Type: buildscripts'
+    files:
+      - '*.yml'
+    branch:
+      - '/buildscripts\/.+/'
+    title:
+      - '/(?<![\w])((buildscript)|(buildscripts))(?![\w])/i'
```

Comment:
Would we want to add `CI` and `CI/CD` to this list, or are those too generic?

Reply:
I like adding those terms! If it's too generic we can always remove them from the config or refine it to something more specific later